ethers-core/src/types/transaction/eip2718.rs

```diff
@@ -440,6 +512,11 @@ impl Decodable for TypedTransaction {
                 // Optimism Deposited (0x7E)
                 Ok(Self::OptimismDeposited(OptimismDepositedTransactionRequest::decode(&rest)?))
             }
+            #[cfg(feature = "kroma")]
+            Some(x) if x == U64::from(0x7E) => {
+                // Kroma Deposited transaction (0x7E)
```

Comment:
it's minor, let's match optimism's comment..!
```suggestion
                // Kroma Deposited (0x7E)
```